### PR TITLE
Some touchups to wording for CLA sign flow

### LINKED ISSUE #64: Language legal review

### LINKED ISSUE #71: Create email template for corporate CLA

### PR DESCRIPTION
**Pull request by bocon13**
closes #64 
closes #71

### CODE REVIEW

File: client/src/js/agreement/CreateAgreementContainer.jsx [hunk 1]
```diff
@@ -31,14 +31,14 @@ function CreateAgreementContainer () {
         <Grid item xs={12} md={6} className={classes.cell}>
           <Link to={`/sign/${AgreementType.INDIVIDUAL}`}>
             <Button variant='contained' color='primary' size='large'>
-              Individual Contributions
+              Only Yourself
```
Comment (teone) [inline client/src/js/agreement/CreateAgreementContainer.jsx]: Bill asked to change `Only yourself` and `Your Employer` to `Individual/Corporate contributions`

> change the button “Yourself” to “Individual Contributions”
> change the button “Your employer” to “Corporate Contributions”

File: client/src/js/helpers/SignCheck.jsx [hunk 2]
```diff
@@ -25,10 +26,9 @@ const useStyles = makeStyles(theme => ({
 function EmailBody (props) {
   return (
     <div>
-      <h2>Copy this email and send it to someone who can sign</h2>
-      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt enim illum, veniam autem excepturi quibusdam
-        ratione iste fuga suscipit cupiditate, dicta perferendis dolor eum minus quaerat velit. Rem, dignissimos
-        repudiandae?</p>
+      <h2>Please email the following to someone at your company who can sign the CLA:</h2>
+      <p><i>If you don't know who that is you can start with your manager or director.</i></p>
+      <ReactMarkdown source={ClaRequestCorporate} />
```
Comment (teone) [inline client/src/js/helpers/SignCheck.jsx]: You'll need `import ReactMarkdown from 'react-markdown'`

File: client/src/js/helpers/SignCheck.jsx [hunk 3]
```diff
@@ -49,7 +49,11 @@ function SignCheck (props) {
 
   return (
     <Paper elevation={23} className={classes.root}>
-      <h2 className={classes.h2}>Can you sign for you company?</h2>
+      <h2 className={classes.h2}>Are you authorized to sign legal agreements for your company?</h2>
+      <p>
+        Typically, this means you are an executive (e.g. CEO or CTO) or officer that explicitly
```
Comment (teone) [inline client/src/js/helpers/SignCheck.jsx]: Typically, this means you are an executive (e.g. CEO or CTO) or officer that  **is** explicitly authorized by your company's board.

File: client/src/js/agreement/CreateAgreementContainer.jsx (hunk 1 above)
Comment (ccascone) [reply client/src/js/agreement/CreateAgreementContainer.jsx]: How about
```suggestion
              Only Yourself / Sign Individual CLA
```

File: client/src/js/agreement/CreateAgreementContainer.jsx [hunk 4]
```diff
@@ -31,14 +31,14 @@ function CreateAgreementContainer () {
...
             </Button>
           </Link>
         </Grid>
         <Grid item xs={12} md={6} className={classes.cell}>
           <Link to={`/sign/${AgreementType.CORPORATE}`}>
             <Button variant='contained' color='primary' size='large'>
-              Corporate Contributions
+              Your Employer
```
Comment (ccascone) [inline client/src/js/agreement/CreateAgreementContainer.jsx]:
```suggestion
              Your Employer / Sign Corporate CLA
```

File: client/src/js/agreement/CreateAgreementContainer.jsx (hunk 1 above)
Comment (teone) [reply client/src/js/agreement/CreateAgreementContainer.jsx]: works for me

**Pull request merged**